pkg/util/kubeobjects/secret/secret.go

```diff
@@ -48,6 +48,18 @@ func (query Query) Update(secret corev1.Secret) error {
 	return query.update(secret)
 }
 
+func (query Query) Delete(name, namespace string) error {
+	query.Log.Info("updating secret", "name", name, "namespace", namespace)
+	tmp := &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: namespace}}
+
+	err := query.KubeClient.Delete(query.Ctx, tmp)
+	if k8serrors.IsNotFound(err) {
+		return nil
+	}
```

Comment:
Maybe add a log here?

Reply:
What about this previous log then
```
query.Log.Info("updating secret", "name", name, "namespace", namespace)
```

Reply:
well that is nice catch 😅
on it 😉

Reply:
<img src="https://media4.giphy.com/media/pcKnpFrumIM7TtzayE/giphy.gif"/>